CMakeLists.txt

```diff
@@ -178,6 +178,12 @@ endif()
 
 find_package(ZLIB)
 
+if ( EXISTS ${OEM_THEME_DIR}/OEM.cmake )
+    set(ICON_APP_NAME ${APPLICATION_SHORTNAME})
+else()
+    set(ICON_APP_NAME "owncloud")
+endif()
+
```

Comment:
Sorry for the delay. I basically used the same as in here: https://github.com/owncloud/client/blob/master/src/gui/CMakeLists.txt#L226-L232

But I guess we could also move to the change you proposed and add the APPLICATION_ICON_NAME to the OWNCLOUD.cmake file.